packages/@atjson/editor/src/webcomponent-renderer/index.ts

```diff
@@ -0,0 +1,93 @@
+import { HIR } from '@atjson/hir';
```

Comment:
OOOOO this isn't a duplicate; it's just an incremental updating WebComponentRenderer?